scripts/src/main/resources/scripts/command/kotlin_native

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+# autocompletion list
+if [ "${1}" = "shortlist" ]
+then
+  if [ -z "${2}" ]
+  then
+    echo "setup version help"
+  fi
+  exit
+fi
+
+# shellcheck source=scripts/functions
+source "$(dirname "${0}")"/../functions
+TOOL_VERSION_COMMAND="kotlinc-native -version"
+# shellcheck source=scripts/commandlet-cli
+source "$(dirname "${0}")"/../commandlet-cli
+
+# $1: optional silent mode
+function doSetup() {
+  doInstall "kotlin_native" "${KOTLIN_NATIVE_VERSION}" "${1}"
+  return 0
+}
+
+# CLI
+case ${1} in
+"help" | "-h")
+  echo "Install kotlin native."
+  echo
+  echo "Arguments:"
+  echo " setup                          install kotlin native on your machine."
+  echo " «args»                         call kotlinnative with the specified arguments. Call kotlinnative --help for details or use kotlinnative directly as preferred."
+;;
+"setup" | "s" | "")
+  doSetup "${2}"
+;;
+*)
+  doRun "${@}"
```

Comment:
function `doRun` does not exist and needs to be added.

Reply:
doRun is added